src/loaders/lottie/tvgLottieModel.h

```diff
@@ -343,6 +343,13 @@ struct LottieText : LottieObject, LottieRenderPooler<tvg::Shape>
         LottieScalar anchor{};
     } alignOption;
 
+    struct
```

Comment:
This is an unusual special feature. You might be able to combine the main TextFollowPath in the Lottie builder you introduced with this way:
```
LottieTextFollowPath* followPath = nullptr;
```

Reply:
I'll apply this